src/common/models/ldap.go

```diff
@@ -24,6 +24,7 @@ type LdapConf struct {
 	LdapUID               string `json:"ldap_uid"`
 	LdapScope             int    `json:"ldap_scope"`
 	LdapConnectionTimeout int    `json:"ldap_connection_timeout"`
+	LdapVerifyCert        bool   `json:"ldap_verify_cert"`
```

Comment:
Duplicate of LDAP struct defined in common/models/config.go

Reply:
Will do it in refactor ldap code

Reply:
the LDAP struct hold the configure values returned to UI, and the LdapConfig hold the value in internal configure and http request, their json format is different.